### ISSUE TITLE
Add option to pass input and receive output

### ISSUE THREAD

**Issue by kaieberl** [namtranase/gemma-cpp-python#4]
Hi again. 
For my project, I would like to send prompts and receive answers from within Python.
Would it be possible to add that to the roadmap?
Thank you in advance.

**Comment by namtranase** [comment]
Yes, I have a plan to do it, will be back when I'm done. Thank you for suggesting this feature!

**Comment by namtranase** [comment]
We have figured out how to do it and @bachvudinh is working on it, we will update you soon.

**Comment by kaieberl** [comment]
Thank you for the update!
In the meantime, it also works using subprocess, although it does not support multi-turn conversations:
```
result = subprocess.run(
    [GEMMA_CPP_PATH, '--', '--tokenizer', '/path/to/gemma.cpp/build/tokenizer.spm',
     '--compressed_weights', '/path/to/gemma.cpp/build/2b-it-sfp.sbs', '--model', '2b-it',
     '--verbosity', '0'],
    input='some input',
    capture_output=True,
    text=True
)
```

**Comment by namtranase** [comment]
We just pushed a new version that supports the completion function. You can check the `tests/test_chat.py` for more details.
In the mean time, we are doing the refactor repo so it can support more features in the future.
